cmd/observe/flows.go

```diff
@@ -200,17 +200,79 @@ func newFlowsCmdWithFilter(vp *viper.Viper, ofilter *flowFilter) *cobra.Command
+  filter for the used HTTP method using the '--http-method' flag. The following
+  command shows all flows that use the HTTP PUT method.
+
+    hubble observe --http-method put
+
+  You can also provide multiple values for a flag, in which case a flow matches the
+  filter if it matches any of the provided values. For our example if we add a second
+  '--http-method' flag matching GET requests, the command will show any flow that
+  uses either a PUT or GET method.
+
+    hubble observe --http-method put --http-method get
+
+  If we add a different flag, a flow is only returned if it matches for both of the
+  different flags. For example we can add a '--to-namespace' flag to our command so
+  only flows using GET or PUT requests to endpoint in namespace 'foo' are returned.
+
+    hubble observe --http-method put --http-method get --to-namespace foo
+
+  And by adding another '--to-namespace' flag, it will return flows using GET or PUT
+  requests to endpoint in namespace 'foo' or 'bar'
+
+    hubble observe --http-method put --http-method get --to-namespace foo --to-namespace bar
+
+* Using reversed filters
+
+  Observe can also return all flows that don't match a certain filter by using the
+  '--not' flag. The following command returns all flows that don't use the HTTP PUT
+  method.
+
+    hubble observe --not --http-method put
+
+  Similarly to filtering for multiple values, we can also filter our multiple values.
+  The following command will return all flows that neither use a GET nor PUT method.
+
+    hubble observe --not --http-method put --not --http-method get
+
+  We can also filter using multiple reversed flags. This will then filter out all
+  flows that match both of the flags. For example, by adding a
+  '--not --to-namespace foo' flag, the example command will show all flows that
+  don't use a GET or PUT method and end in the 'foo' namespace.
+
+    hubble observe --not --http-method put --not --http-method get --not --to-namespace foo
+
+  This means the command will still return flows to the 'foo' namespace that don't
+  use HTTP PUT or GET methods, and it will return flows using HTTP PUT and GET
+  methods that end in other namespaces.
+  `,
 		use:   "flows",
 		short: "Observe flows of a Hubble server",
 		long: `Observe provides visibility into flow information on the network and
 application level. Rich filtering enable observing specific flows related to
 individual pods, services, TCP connections, DNS queries, HTTP requests and
-more.`,
+more.
+
+Observe can filter flows using multiple different flags, such as filtering for
+the source namespace or the used protocol. To match a filter flag, a flow most
+match at least one of the provided values for that flag. A flow must match all
+the provided filter flags to be returned.
+
+Observe can also show all flows that do not match a provided filter, by adding
+the '--not' flag in front of a filter flag. To not match a reversed filter
```

Comment:
Reversed -> negated and remove double negative.
```suggestion
the '--not' flag in front of a filter flag. To match a negated filter
```